daemon/daemon.go

```diff
@@ -66,16 +67,30 @@ func (ib *Infoblox) Allocate(args *ExtCmdArgs, result *types.Result) (err error)
 
 	ipn, _ := types.ParseCIDR(subnet)
 	ipn.IP = net.ParseIP(ip)
-	result.IP4 = &types.IPConfig{
-		IP:      *ipn,
+	ipConfig := &current.IPConfig{
+		Version: "4",
+		Address: *ipn,
 		Gateway: conf.IPAM.Gateway,
-		Routes:  conf.IPAM.Routes,
 	}
+	routes := convertRoutesToCurrent(conf.IPAM.Routes)
```

Comment:
Just wondering.... so, there's not already an utility in CNI that that does this conversion already?